store/src/java/com/zimbra/cs/db/DbPool.java

```diff
@@ -382,6 +382,75 @@ public static DbConnection getConnection(Mailbox mbox) throws ServiceException {
         }
     }
 
+    public static DbConnection getConnection(Integer mboxId, Integer schemaGroupId) throws ServiceException {
```

Comment:
Where is this new method being used?

Reply:
This is used in IndexingService (specifically, IndexingTask::run), where the Mailbox object is not available.